Unity/Assets/Cars/SteeringWheelSimCar/elements/RayCasts.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+public class RayCasts : MonoBehaviour
+{
+    // length of raycasts
+    public float length = 10;
+
+    [Range(1, 360)] public float angle = 120;
+
+    public int numberOfRays = 10;
+
+    private void FixedUpdate()
```

Comment:
outsource drawing code to new method "DrawRayCasts()" and call it in FixedUpdate(), also add public bool variable "drawRays" to enable turning off raycast drawing